public/stylesheets/index.css

```diff
@@ -232,3 +232,17 @@ span.copy-success-icon {
   width: 16px;
   height: 16px;
 }
+
+.field label {
+  width: 10em;
+  margin-top: 15px;
+}
+
+.field .pacticipant_name {
+  width: 20em;
+}
+
+div.top-of-group {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
```

Comment:
When I test it locally, looks like the change in the `index.css` file does not
get reflected
Copy those changes and put into Firefox css style editor result in the image in
the PR description

Im not sure what I did wrong which made the `index.css` changes not get copied
to the server asset. Any hint?

Reply:
![image](https://user-images.githubusercontent.com/4962863/120826987-54043a80-c59e-11eb-8323-fe6e6e6f336b.png)

Reply:
Hm, I can't think of anything to be honest.

Reply:
It is fixed now. Im not sure why it did not work before :shrug: